examples/core/core_3d_camera_first_person.c

```diff
@@ -25,13 +26,14 @@ int main(void)
 
     InitWindow(screenWidth, screenHeight, "raylib [core] example - 3d camera first person");
 
-    // Define the camera to look into our 3d world (position, target, up vector)
-    Camera camera = { 0 };
-    camera.position = (Vector3){ 4.0f, 2.0f, 4.0f };
-    camera.target = (Vector3){ 0.0f, 1.8f, 0.0f };
-    camera.up = (Vector3){ 0.0f, 1.0f, 0.0f };
-    camera.fovy = 60.0f;
-    camera.projection = CAMERA_PERSPECTIVE;
+    // Define the camera to look into our 3d world
+    Camera3D camera = {0};
+    InitializeCamera(&camera, (double)screenWidth / (double)screenHeight);
```

Comment:
I prefer current approach, exposing the standard camera parameters to the user directly instead of hiding them inside an opaque `InitCamera()`.

Reply:
Sure. I agree it is nicer for the user to see everything directly.
I'll remove the function and adjust this in the example.